.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,48 +1,48 @@
-name: "\U0001F41E Bug report"
```

Comment:
Not sure how I feel about yaml formatting, doesn't really change much as far as readability is concerned IMHO